README.md

```diff
@@ -75,6 +80,38 @@ end
 
 ```
 
+```ruby
+Register Listening Tentacle
+# You will first need to generate an api key
+# In Octopus Deploy Server GUI click your Name -> Profile -> API keys
+octopus_deploy_tentacle 'Tentacle' do
+  action :register
+  server 'https://octopus.example.com'
+  api_key '12345678910'
+  roles ['database']
+  environment 'prod'
+  polling false
```

Comment:
For listening tentacles we do not need the polling boolean or the port specified.  So for this example we can actually just leave those attributes off.  We only need to add the values if its  not the defaults.